### ISSUE TITLE
Import bookmarks with other services

### ISSUE THREAD

**Issue by jocelyn** [hoarder-app/hoarder#322]
I am looking at a way to import a collection of bookmarks into hoarder app.
So far, I found 2 solutions
1) from the webUI, dropping a HTML bookmarks file
2) using the command line tool

But in both cases, I don't see how to import the tags.
I understand the webUI import feature only import the "url", but ignore all other values (such as title, description, tags, dates, ...)

Using the CLI, it is a bit better, as one can
- set the title, note, archived or not, favourite or not
- it is possible to add a bookmark to a list

But I don't see how to set tags on bookmarks from the CLI.

This is the only missing feature for me.

With it, one could import from CLI, any collection of tagged bookmarks (with custom script).
And it would also be possible to export using the CLI the current hoarder-app data.

Would it be possible to add:
```
  hoader tags add "new-tag"
  hoader tags add-bookmark --bookmark <bookmark> --tag <tag-id> 
```  

The process would not be fast, but at least possible.

**Comment by MohamedBassem** [comment]
#211 adds support for attaching tags to bookmarks :) Will review it soon.

**Comment by MohamedBassem** [comment]
#211 got merged, in CLI version `0.13.7`, you'll be able to do `hoarder bookmarks update-tags --add-tag <tag>`
